.github/workflows/nightly.yml

```diff
@@ -58,3 +58,65 @@ jobs:
     - name: Fuzz long test
       working-directory: ${{github.workspace}}/build
       run: ctest -C ${{matrix.build_type}} --output-on-failure -L "fuzz-long"
+
+  adapter-build:
+    name: Build - Adapters on Ubuntu
+    strategy:
+      matrix:
+        adapter: [L0]
+        build_type: [Debug, Release]
+        compiler: [{c: gcc, cxx: g++}, {c: clang, cxx: clang++}]
+
+    runs-on: self-hosted
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install pip packages
+      run: python3 -m pip install -r third_party/requirements.txt
+
+    - name: Download DPC++
+      run: |
+        wget -O ${{github.workspace}}/dpcpp_compiler.tar.gz https://github.com/intel/llvm/releases/download/sycl-nightly%2F20230626/dpcpp-compiler.tar.gz
+        tar -xvf ${{github.workspace}}/dpcpp_compiler.tar.gz
+
+    - name: Configure CMake
+      run: |
+        export CPATH=/usr/include/c++/11/:/usr/include/x86_64-linux-gnu/c++/11/
+        export LIBRARY_PATH=/usr/lib/gcc/x86_64-linux-gnu/11:/home/u105190/sycl_debug_ws:$LIBRARY_PATH
```

Comment:
heh, you can't use such hardcoded paths (that may change), like this: `/home/u105190/sycl_debug_ws`.

Please use some kind of other vars to make this happen differently - e.g.:
 - GHA's variable `${{github.workspace}}`
 - environment variables like `$HOME`, or
 - anything else